templates/prometheus.yaml

```diff
@@ -32,8 +32,8 @@ spec:
       enabled: true
   chartReference:
     chart: prometheus-operator
-    repo: https://mesosphere.github.io/charts/staging
-    version: 5.10.9
+    repo: https://gracedo.github.io/charts/staging
```

Comment:
Let's just make sure we bump this back to mesosphere when we're unblocked before merge :+1: